shared-ui/components/footer/Footer.styles.ts

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { min, max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../style/colors';
+import { P, fonts } from '../../style/typography';
+
+
+const StyledFooter = styled.div`
+  position: absolute;
+  width: 100%;
+  background-color: ${colors.HEADER_FOOTER_BLUE};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 23em;
+  @media ${min.tablet} {
+    flex-direction: row;
+    align-items: flex-start;
+  }
+`
+
+const StyledLogoContainer = styled.div`
+  flex-basis: 20%;
+`
+
+const StyledHackBeanpotLogo = styled.img`
+  padding: 0.7em;
+`
+
+const StyledTabContainer = styled.div`
+  @media ${max.tablet} {
+    display: none;
+  }
+  @media ${max.desktop} {
+    padding: 8em 0em 8em 0em;
+  }
+  height: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-self: center;
+  flex-basis: 40%;
+  padding: 8em 0 8em 10em;
+`
+
+const StyledTab = styled(P)`{
+  color: ${colors.WHITE};
+  font-family: ${fonts.nunitoSansSemibold};
+  padding: 0.15em;
+}
+`;
+
+const StyledButtonContainer = styled.div`
+  height: 100%;
+  box-sizing: border-box;
+  flex-basis: 40%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  @media ${max.tablet} {
+    padding: 0;
+    width: 20em;
+  }
+  @media ${max.desktop} {
+    align-items: center;
+    gap: 12px;
+    justify-content: center;
+  }
+  padding: 3em 0;
+`
+
+const StyledEmailInput = styled.input`
```

Comment:
Can we also add a max-width of maybe like 28em? On larger screens the input looks a bit too wide LOL